samples/azure-quantum/hidden-shift/hidden-shift.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Quantum algorithms for fast convolutions and hidden shifts\n",
+        "\n",
+        "Welcome, this notebook explores the execution of _hidden shift problems_ on a quantum computer.\n",
+        "This notebook does assume some quantum knowledge.  We invite you to have a look into our 👋🌎 _Hello world_ samples in the notebook gallery and the [Quantum Katas](https://docs.microsoft.com/azure/quantum/tutorial-qdk-intro-to-katas) for introductory content on quantum computing.\n",
```

Comment:
The "notebook gallery" is not a well-defined term outside of the context of the Azure Quantum blade of the Azure Portal.

```suggestion
        "This notebook assumes some quantum knowledge.  We invite you to have a look into our 👋🌎 [_Hello world_ samples](https://docs.microsoft.com/samples) (available in the Azure Portal notebook gallery) and the [Quantum Katas](https://docs.microsoft.com/azure/quantum/tutorial-qdk-intro-to-katas) for introductory content on quantum computing.\n",
```

Can we deep link to that blade, even if not to the specific notebook?

I'd also suggest linking to the concepts guides in the AQ docs for more beginner-friendly intro QC content.

Reply:
It's not possible to deep link to the blade, since the link requires the workspace resource id and typically a tenant hint, which changes by the user.

We originally were going to add relative link here the samples gallery, but it breaks locally and also breaks link scanning in this repo.

Reply:
I added more explicit language for samples gallery in Azure Quantum workspaces vs the Microsoft Samples Gallery (and linked to that specifically).